scanner/src/routes/+page.svelte

```diff
@@ -394,6 +395,7 @@
     color: white;
     font-size: 1.2rem;
     font-family: "Poppins", sans-serif;
```

Comment:
It will be nice to have separate files with global styles (where fonts go to) and variables somewhere in `src/lib/styles`